### ISSUE TITLE
.gitignore installed by installer breaks application when shared

### ISSUE THREAD

**Issue by emil-nasso** [symfony/symfony-installer#217]
When installing symfony using `symfony new teststuff` it doesn't seem to use the .gitignore-file that can be found here: https://github.com/symfony/symfony-standard/blob/master/.gitignore

Instead it puts this into the file:

```
/app/config/parameters.yml
/bin/
/build/
/composer.phar
/vendor/
/web/bundles/
/var/
!var/cache/.gitkeep
!var/logs/.gitkeep
!var/sessions/.gitkeep
/phpunit.xml
```

This will break sharing of the application because the `bin` directory is ignored.

To reproduce:

* `symfony new test`
* `cd test; git init; git push {somewhere}`
* `cd /tmp; git clone {somewhere}; cd test`
* `composer install`

Composer will exit with errors because it can't find `app/console` or `bin/console`. It can be fixed by replacing the `.gitignore` with the one from symfony standard edition and commit/push the files from the original repository.

**Comment by ogizanagi** [comment]
I agree the fact `.gitignore` are not in sync is an issue. That's why I opened #211

**Comment by emil-nasso** [comment]
@ogizanagi yeah. I agree. And in this case, it actually breaks the application.

**Comment by baiolo** [comment]
Behaviour confirmed.
Temporary workaround:

- Remove the installer .gitignore and replace with the one form [https://github.com/symfony/symfony-standard/blob/master/.gitignore](https://github.com/symfony/symfony-standard/blob/master/.gitignore).

- Copy the `bin` folder from a freshly installed Symfony project into the shared project.

**Comment by ogizanagi** [comment]
This will be fixed once #218 is merged, and a new version of the installer released.

In the meanwhile, you can test this patch by building the phar yourself, or download it from this fake release on Github: https://github.com/ogizanagi/symfony-installer/releases/tag/synced_gitignore

**Comment by stof** [comment]
Closing as #218 has been merged, and released in 1.5.0
